### ISSUE TITLE
Blows up (SyntaxError) in Python 3.7+

### ISSUE THREAD

**Issue by uogbuji** [zepheira/pybibframe#24]
If you try to run pybf in the latest Python versions you will get tracebacks ending with:

```
  File "/Users/uche/.local/pyenv/zdev/lib/python3.7/site-packages/bibframe/reader/engine.py", line 171
    task = asyncio.async(wrap_task(), loop=loop)
                       ^
SyntaxError: invalid syntax
```

The poor asyncio design has finally caught up with us. The fact that `async` is now a keyword, which is the root cause here is just one of the many issues. I'm working on a core refactoring I've wanted to do for over a year.

**Comment by uogbuji** [comment]
This is fixed on the develop branch (dbb533c) but I have some things to check before making a full release. If you happen to be blocked on this, though, just switch branches.

**Comment by uogbuji** [comment]
The full release 1.1.1 should address this issue.
